model/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -144,6 +144,16 @@ case class WorkflowCost(
   cost: Option[Float]
 )
 
+case class WorkflowCostBreakdownParams(includeTaskBreakdown: Boolean, includeSubworkflowBreakdown: Boolean)
+
+case class WorkflowCostBreakdown(id: String,
+                                 cost: BigDecimal,
+                                 currency: String,
+                                 status: String,
+                                 subworkflowBreakdown: Option[Map[String, BigDecimal]],
```

Comment:
These two breakdown items will not be included after all, but there will be a list of strings with the key `errors` that describes any issues we ran into while computing the cost. Might be worth having a separate conversation about the various error conditions and how we want to handle them, but if `errors` is nonempty we know that the `cost` is NOT the full current cost of the workflow.